domains/domains.go

```diff
@@ -263,8 +263,8 @@ type Repository interface {
 	// UpdateRejection updates an invitation by setting the rejection time.
 	UpdateRejection(ctx context.Context, invitation Invitation) (err error)
 
-	// Delete deletes an invitation.
-	DeleteInvitation(ctx context.Context, userID, domainID string) (err error)
+	// DeleteUsersInvitation deletes users invitation associated with a domain.
+	DeleteUsersInvitation(ctx context.Context, domainID string, userID ...string) (err error)
```

Comment:
Use plural, `DeleteUsersInvitations`. Also, update the comment to something like: `DeleteUsersInvitations deletes invitation to a provided domain for users with provided user IDs.`